python/eos/data/native.py

```diff
@@ -18,6 +18,7 @@
 import numpy as _np
 import pypmc
 import yaml
+import dynesty
```

Comment:
@dvandyk do we make `dynesty` an EOS dependence as we did for `pypmc`? Otherwise this needs to be moved to the corresponding function and protected.

Reply:
@fnovak42 can you please adjust the tags for the two docker images in ``.github/workflows``, in the Ubuntu and Manylinux2014 worksflows? They should now point to ``c3b826a70d729805258243fb5d95dbe5ae2fbd2f``, which will be available in ~10min from now.